src/test/java/engineering/swat/watch/TestHelper.java

```diff
@@ -48,9 +49,21 @@ else if (os.contains("win")) {
             // especially on small core systems
             delayFactor *= 4;
         }
+        TINY_WAIT = Duration.ofMillis(250 * delayFactor);
         SHORT_WAIT = Duration.ofSeconds(1 * delayFactor);
         NORMAL_WAIT = Duration.ofSeconds(4 * delayFactor);
         LONG_WAIT = Duration.ofSeconds(8 * delayFactor);
     }
 
+    public static void trySleep(Duration duration) {
+        trySleep(duration.toMillis());
+    }
+
+    public static void trySleep(long millis) {
+        try {
+            Thread.sleep(millis);
+        } catch (InterruptedException e) {
```

Comment:
[Never ignore/wrap an interrupted exception](https://www.baeldung.com/java-interrupted-exception). The best thing to do is to call:

```
Thread.currentThread().interrupt();
```

after we're done cleaning up.